Internals/Events/channelDelete/GAB.ChannelDelete.js

```diff
@@ -0,0 +1,97 @@
+const BaseEvent = require("../BaseEvent.js");
+
+class ChannelDelete extends BaseEvent {
+	requirements (channel) {
+		return Boolean(channel.guild);
+	}
+
+	async handle (channel) {
+		const serverDocument = await Servers.findOne(channel.guild.id);
+		if (!serverDocument) {
+			return logger.debug("Failed to find server data for channel deletion.", { svrid: channel.guild.id, chid: channel.id });
+		}
+		const serverQueryDocument = serverDocument.query;
+
+		let updated = false;
+		const channelDocument = serverDocument.channels[channel.id];
+		if (channelDocument) {
+			updated = true;
+			serverQueryDocument.clone.id("channels", channel.id).remove();
+		}
+
+		// Command channel configurations
+		for (const command in serverDocument.config.commands) {
```

Comment:
Sweeet jesus no, please for-of Object.entries

Reply:
Fine, I'll bite. HoW sO?

Reply:
```js
for (const [key, command] of Object.entries(serverDocument.config.commands))
```

Reply:
no i know how it works you dingus